drivers/all_neighbors_system_dashboard/app/models/all_neighbors_system_dashboard/enrollment_attribute_calculations.rb

```diff
@@ -116,66 +116,48 @@ def race_list(enrollment)
       end
 
       def ce_infos_for_batch(filter, batch)
-        household_enrollments = household_enrollments(filter, batch)
-        ce_events = ce_events(filter, household_enrollments)
+        # Find the active enrollments in the CE Project set for the HoH of the enrollments in the batch
+        ce_project_enrollments = GrdaWarehouse::ServiceHistoryEnrollment.
+          entry.
+          where(project_id: GrdaWarehouse::Hud::Project.where(id: filter.secondary_project_ids).pluck(:project_id)).
```

Comment:
I believe SHE.project_id is actually Project.ProjectID

Reply:
Confirmed.  You can `join(:project).merge(GrdaWarehouse::Hud::Project.where(id: filter.secondary_project_ids).pluck(:project_id))`  what you have might work, but will have some issues with the data_source_id in the resulting data.

Reply:
Yes --but `filter.secondary_project_ids` returns Rails ids... Maybe that should change?